lib/Analysis/AxisInfo.cpp

```diff
@@ -467,6 +467,13 @@ class DivOpAxisInfoVisitor final : public BinaryOpVisitorImpl<OpTy> {
     if (rhs.getConstantValue().has_value() &&
         rhs.getConstantValue().value() == 1)
       return lhs.getDivisibility(dim);
+    // Case 3: lhs has contiguity of 1 in this dimension and rhs is a power of 2
```

Comment:
It makes me think of the other scenario in which rhs has the contiguity of 1.

For example, 8 / [2, 4, 2] => [4, 2, 4]

Reply:
Yeah, you're right. Thanks for correcting me